docs/UserGuide.md

```diff
@@ -89,9 +89,10 @@ Examples:
 
 ### Listing all persons : `list`
 
-Shows a list of all persons in the address book.
+Lists the existing contacts based on the group. 
+If the group name is unspecified, then list all the existing contacts.
 
-Format: `list`
+Format: `list [g/GROUP_NAME]`
```

Comment:
Similarly, I have decided to drop this feature as it seems to be the same either with 'find'/'track' command. So, it will just be `list` instead of `list [g/GROUP_NAME]`.